src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,36 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
```

Comment:
Looks like `exfmt` can be switched to using `mix`'s `format` command: https://hexdocs.pm/mix/Mix.Tasks.Format.html 🎉

Reply:
A good idea, I've deprecated exfmt 👍